ehr/src/org/labkey/ehr/EHRServiceImpl.java

```diff
@@ -443,7 +443,7 @@ public void registerOptionalClinicalHistoryResources(Module module)
         EHRService.get().registerHistoryDataSource(new DefaultArrivalDataSource(module));
         EHRService.get().registerHistoryDataSource(new DefaultAssignmentEndDataSource(module));
         EHRService.get().registerHistoryDataSource(new DefaultBirthDataSource(module));
-        EHRService.get().registerHistoryDataSource(new DefaultClinicalRemarksDataSource(module));
+       // EHRService.get().registerHistoryDataSource(new DefaultClinicalRemarksDataSource(module));
```

Comment:
To remove this here, we need to add it to WNPRC_EHRModule.doStartupAfterSpringConfig() so they don't lose it.

Also, let's delete the line instead of commenting it out

Reply:
done